### PR TITLE
Use kairos-release instead of os-release

### PR DESCRIPTION
**Pull request by Itxaka**
Part of https://github.com/kairos-io/kairos/issues/2907

### CODE REVIEW

**Review by jimmykarily — APPROVED** [review]

**Comment by mudler** [comment]
this needs to wait after the patch release is out. just making sure we don't merge it accidentally

**Review by mudler — DISMISSED** [review]
Good to merge after the patch release

**Pull request merged**
